thor/src/lib.rs

```diff
@@ -57,18 +55,6 @@ use bitcoin::util::amount::serde::as_sat;
 /// Flat fee used for all transactions involved in the protocol, in satoshi.
 pub const TX_FEE: u64 = 10_000;
 
-#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-#[derive(Clone, Debug)]
-pub struct Channel {
-    x_self: OwnershipKeyPair,
-    X_other: OwnershipPublicKey,
-    final_address_self: Address,
-    final_address_other: Address,
-    tx_f_body: FundingTransaction,
-    current_state: ChannelState,
-    revoked_states: Vec<RevokedState>,
-}
-
 #[async_trait]
```

Comment:
I left `MedianTime` and `GetRawTransaction` in `lib.rs` since they are to do with a 'connector' and not specific to channels. Can change if you disagree.